content/en/docs/concepts/cluster-administration/node-shutdown.md

```diff
@@ -265,6 +265,28 @@ via the [Non-Graceful Node Shutdown](#non-graceful-node-shutdown) procedure ment
 {{< /note >}}
 
 
+## Windows Graceful node shutdown {#windows-graceful-node-shutdown}
+
+{{< feature-state feature_gate_name="WindowsGracefulNodeShutdown" >}}
+
+The Windows graceful node shutdown feature depends on kubelet running as a Windows service, 
+it will then have a registered [service control handler](https://learn.microsoft.com/en-us/windows/win32/services/service-control-handler-function) 
+to delay the  presshutdown event with a given duration.
+
+Windows graceful node shutdown is controlled with the `WindowsGracefulNodeShutdown` 
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) 
+which is introduced in 1.32 as an alpha feature.
```

Comment:
Hi @zylxjtu  If a user wants to use this feature, the GracefulNodeShutdown and WindowsGracefulNodeShutdown feature gate must be turned on together. Is it correct?

Reply:
https://github.com/carlory/kubernetes/blob/master/pkg/kubelet/apis/config/validation/validation.go#L178

Reply:
no, depends on which platform are you on, you only need to enable one of them

Reply:
That sounds like a bug. If `GracefulNodeShutdown` is false, the kubelet shouldn't provide graceful node shutdown.

(or, we should document that the feature gate is misnamed and ought to be treated as if it were named `LinuxGracefulNodeShutdown`).